modules/repository-datastax/db/mutation/V031__Drop_User_Experiment_Index_table.cql

```diff
@@ -0,0 +1 @@
+drop table user_experiment_index;
```

Comment:
Would it be better to use "DROP TABLE IF EXISTS keyspace_name.table_name" syntax?